CHANGELOG.md

```diff
@@ -11,10 +11,6 @@ All notable changes to this project will be documented in this file based on the
 
 * Remove `expected_values` from `threat.*.indicator.name` fields. #2281
 
-#### Added
```

Comment:
Alternatively, you could forward-port the commit from https://github.com/elastic/ecs/pull/2303/files into `main`. But not absolutely necessary.

Reply:
Good call out, I'll switch to that